pkg/core/resources/apis/meshservice/status/updater_test.go

```diff
@@ -230,10 +230,10 @@ var _ = Describe("Updater", func() {
 				Total:     3,
 			},
 		}),
-		Entry("should count healthy DPPs", dpProxiesTestCase{
+		Entry("should count healthy DPPs and use MeshService.ports[].targetPort to select DP inbound", dpProxiesTestCase{
 			meshService: samples.MeshServiceBackendBuilder().
 				WithDataplaneTagsSelectorKV("app", "backend").
-				AddIntPort(builders.FirstInboundPort+1, builders.FirstInboundServicePort+1, core_mesh.ProtocolHTTP),
```

Comment:
The positions of parameters was reversed in previous code: service port should be the first argument, and DP port is  the second.

Reply:
`MeshServiceBuilder` Sorce: https://github.com/kumahq/kuma/blob/release-2.10/pkg/test/resources/builders/meshservice_builder.go#L67-L77